### PR TITLE
Document reactive health check

### PR DESCRIPTION
**Pull request by loicmathieu**

### CODE REVIEW

File: docs/src/main/asciidoc/smallrye-health.adoc
Comment (Ladicek) [inline docs/src/main/asciidoc/smallrye-health.adoc:329]:
```suggestion
MicroProfile Health currently doesn't support returning reactive types, but SmallRye Health does.
```

Just a little language simplification.

**Comment by Ladicek** [comment]
Looks good to me. Maybe @xstefank should review?

**Comment by loicmathieu** [comment]
@Ladicek I apply your suggestions.

**Comment by loicmathieu** [comment]
@Ladicek can I have your approval so I can merge this one?

**Comment by Ladicek** [comment]
I hoped that @xstefank would review, but...

**Review by Ladicek — APPROVED** [review]

**Comment by loicmathieu** [comment]
Thanks @Ladicek he can still have a look and propose improvements later.

File: docs/src/main/asciidoc/smallrye-health.adoc
Comment (xstefank) [inline docs/src/main/asciidoc/smallrye-health.adoc:346]: I believe that AppScoped is not required because quarkus makes Liveness a bean-defining annotation, but I can check that later. Nothing big.

**Comment by xstefank** [comment]
LGTM! I guess I can't take PTO @Ladicek ?? :D

**Comment by Ladicek** [comment]
Haha sorry! :-)

**Pull request merged**
